tests/unit/test_data_cleaner.py

```diff
@@ -74,3 +74,29 @@ def test_data_cleaner():
 
     assert len(actual) == len(expected_text)
     assert all([a == b for a, b in zip(actual, expected_text)])
+
+
+def test_clean_text_from_specific_characters(transformed_pwdb_dataframe):
+    pwdb_descriptive_data = transformed_pwdb_dataframe['title'].map(str) + ' ' + \
+                            transformed_pwdb_dataframe['background_info_description'].map(str) + ' ' + \
+                            transformed_pwdb_dataframe['content_of_measure_description'].map(str) + ' ' + \
+                            transformed_pwdb_dataframe['use_of_measure_description'] + ' ' + \
+                            transformed_pwdb_dataframe['involvement_of_social_partners_description']
+    unused_characters = ["\\r", ">", "\n", "\\", "<", "''", "%", "...", "\'", '"', "(", "\n", "[", "]"]
+    clean_text = clean_text_from_specific_characters(pwdb_descriptive_data, unused_characters)
+
+    assert "\\r" not in clean_text
```

Comment:
these lines shall be reduced to a single assert. Investigate how to do it in a pythonic way.

Reply:
perhaps any() or all() functions can be of help

Reply:
I tried to figure out how to reduce it into a single line, but the test expects to return a string not a bool or a list

TypeError: 'in <string>' requires string as left operand, not list -> using the list
TypeError: 'in <string>' requires string as left operand, not bool -> using all() or any()